Automattic/jetpack
# Add/backup plan recommendation

This PR adds a recommendation for backups into the recommendations flow. This recommendation shows up after auto-updates are enabled for a plugin on the site.

#### Changes proposed in this Pull Request:

* This PR adds a new recommendation step for backups in the recommendation flow

#### Other information:


#### Jetpack product discussion


pbNhbs-3jP-p2

#### Does this pull request change what data or activity we track or use?


This PR does not add any new events/ data. Existing events that are already in place for other recommendations will be used to track interactions with this new recommendation.

#### Testing instructions:






* On your Jetpack test site, start by resetting the Jetpack options. If you are using a development version of Jetpack, this option should be available in the footer on the Jetpack Dashboard.
* Navigate to the plugins page and click "Enable auto updates" for any of the plugins on the site. You can also toggle auto-updates off and back on for a plugin that already has the setting enabled.
* Navigate back to the Jetpack dashboard, you should see a red "1" bade next to the "Recommendations" navigation item indicating that a new recommendation is available.
![Screen Shot 2022-08-10 at 11 35 56 AM](https://user-images.githubusercontent.com/18016357/183954260-1c81c060-005d-4afa-9116-f91872146d96.png)

* Click on the recommendations navigation item. You should now see the recommendation for site backups. There should be a green "New" badge showing on this step.
![Screen Shot 2022-08-11 at 3 46 04 PM](https://user-images.githubusercontent.com/18016357/184231371-1e01a9af-f376-47be-9881-ac58721eead1.png)

* Click on the CTA to open the article in a new tab. This should link you to this post on jetpack.com: https://jetpack.com/blog/wordpress-website-backups-101/
* Return to the tab with your test site, the recommendations flow should now be showing the summary screen. You should see "Site Backups" showing in the resources section at the bottom of the summary screen.
![Screen Shot 2022-08-11 at 3 46 17 PM](https://user-images.githubusercontent.com/18016357/184231489-d79ce480-a9bb-4c7e-8971-11a9dcdc1e6f.png)

* Click on the "Read More" button next to the Site Backups recommendation on the summary screen. This should take you to again to the article linked above.
* Repeat the test using a test site that already has a Jetpack backup product or a plan that has backup features. This time, the recommendation should not be enabled after auto-updates are turned on for a plugin.
* Confirm that any new text is properly translated.
* Confirm that all links on the new recommendation step work and make sense.
